### ISSUE TITLE
Duplicated PlayerInput.cs in prefabs disable input in builds for Windows Mixed Reality

### ISSUE THREAD

**Issue by jorgeolothar** [ExtendRealityLtd/Tilia.Input.UnityInputSystem#54]
### Environment

* Code source: Tilia.Input.UnityInputSystem 
* Code version: 2.4.3
* Platform version: Unity 2021.3.24
* 3rd party dependencies: Windows Mixed Reality Portal

### Steps to reproduce

In a simple scene, or in the tilia farm, add the `UnityInputSystem.Poses.GenericXR.prefab` into the scene. Optionally, have the pointers follow the prefab's `LeftPointer` and `RightPointer`  instead of the `LeftControllerAlias` and `RightControllerAlias`. If not there already, add the `UnityInputSystem.InputActionMappings.GenericXR.prefab` and map actions to it.

### Expected behavior

After a build, and using WMR OpenXR, pointers follow the controller pointer pose, like in the editor.

### Current behavior

Controllers are tracked, but no controller input is received.

### Testing done

I tested multiple configurations and builds. It seems that if multiple instances of `PlayerInput.cs` are enabled, the Windows Mixed Reality OpenXR runtime stops processing input. This happens only on builds, and only for WMR. Disabling all but one of the scripts fixes the issue.

I recommend adding a script to both Tilia input prefabs that `OnReset()` checks for enabled instances of `PlayerInput.cs` in the scene hierarchy, and disables it from its own prefab if another instance is active. A debug warning would be useful as well.

**Comment by thestonefox** [comment]
as the PlayerInput.cs script is a Unity component (not part of Tilia), would it not be better to raise this as an official Unity bug?

I feel like this is something Unity should probably address

**Comment by jorgeolothar** [comment]
I was wondering about that as well. Documentation says the `PlayerInput.cs` can be used multiple times for local multiplayer, one script per player, each with enforced different devices. When instancing `Poses.GenericXR.prefab` and `InputActionMappings.GenericXR.prefab` I think it's setting up local multiplayer, per design. Not sure why it causes issues only in WMR.

Can we merge the poses and the input action mapping into a single prefab, with a single instance of `PlayerInput.cs`?

**Comment by thestonefox** [comment]
I'm not sure the poses stuff always works if you're not using openxr, that's why its separate at the moment.

Obviously you can merge them yourself.

As they are samples, they're just there to highlight a possible solution. So I wonder if the VRTK resolution to this is just documentation to explain one needs turning off if you use both.

**Comment by jorgeolothar** [comment]
> I'm not sure the poses stuff always works if you're not using openxr, that's why its separate at the moment.

Makes sense to add it to the documentation. It was a difficult bug to find. Steam rejected my app from their store because of it. Luckily I still had a WMR headset to test! I'd like to spare that pain for other people
